react/react-training/src/index.js

```diff
@@ -1,5 +1,13 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
 import Game from '@screens/Game';
+import { store } from '@redux/store';
 
-ReactDOM.render(<Game />, document.getElementById('root'));
+const App = () => (
```

Comment:
Isn't a good practice to use arrow function for the definition of components. Check this [style guide](https://github.com/Wolox/tech-guides/blob/master/react/style-guide.md#class-vs-reactcreateclass-vs-stateless)